lua/rzls/server/methods/codeactionresolve.lua

```diff
@@ -0,0 +1,27 @@
+local documentstore = require("rzls.documentstore")
+local razor = require("rzls.razor")
+
+---@param params lsp.CodeAction
+---@return lsp.CodeAction | nil
+return function(params)
+    local rvd = documentstore.get_virtual_document(params.data.TextDocument.uri, razor.language_kinds.razor)
+
+    if not rvd then
+        return
+    end
+
+    -- HACK: neovim does not deserialize `null`s and rzls expects an explicit `null` value
```

Comment:
Can you call out the Neovim issue so we can remove this when it's fixed 

https://github.com/neovim/neovim/issues/31368